xwiki/CVE-2024-21650/docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+# ---------------------------------------------------------------------------
+# See the NOTICE file distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+# ---------------------------------------------------------------------------
+version: '2'
+networks:
+  bridge:
+    driver: bridge
+services:
+  # The container that runs XWiki in Tomcat, with the appropriate JDBC driver (for mariadb).
+  web:
+    image: "xwiki:${XWIKI_VERSION}-mariadb-tomcat"
+    container_name: xwiki-mariadb-tomcat-web
+    depends_on:
+      - db
+    ports:
+      - "8080:8080"
+    # Default values defined in .env file.
+    # The DB_USER/DB_PASSWORD/DB_DATABASE/DB_HOST variables are used in the hibernate.cfg.xml file.
+    environment:
+      - XWIKI_VERSION=${XWIKI_VERSION}
+      - DB_USER=${DB_USER}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_DATABASE=${DB_DATABASE}
+      - DB_HOST=xwiki-mariadb-db
+    # Provide a name instead of an auto-generated id for xwiki data (the permanent directory in included in it)
+    # configured in the Dockerfile, to make it simpler to identify in 'docker volume ls'.
+    volumes:
+      - xwiki-data:/usr/local/xwiki
+    networks:
+      - bridge
+  # The container that runs the database (mariadb)
+  db:
+    image: "mariadb:11.1"
+    container_name: xwiki-mariadb-db
+    # - Provide a name instead of an auto-generated id for the mariadb data, to make it simpler to identify in
+    # 'docker volume ls'
+    # - Provide a SQL script to be executed when the db image starts (to set permissions to create subwikis)
+    volumes:
+      - mariadb-data:/var/lib/mysql
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
The `init.sql` file is missing, please provide it as the container fails to start without it

Reply:
I did remove this volume as it was empty